.github/workflows/commit.yml

```diff
@@ -70,6 +70,8 @@ jobs:
         uses: arduino/setup-task@v1
       - name: Initialize the repo
         run: task init
+      - name: Build the goat
+        run: task build
```

Comment:
This fixes [this issue](https://github.com/SeisoLLC/goat/actions/runs/5695183825/job/15437866356) which is because the bump-pre-commit job used the `goat` from docker hub and not from the repo